tests/testHSCColorterms.py

```diff
@@ -0,0 +1,97 @@
+import os
+import numbers
+import unittest
+
+import lsst.utils.tests as utilsTests
+import lsst.pipe.tasks.photoCal as photoCal
+
+class ColortermOverrideTestCase(unittest.TestCase):
+    def setUp(self):
+        """Test that colorterms specific to HSC override correctly"""
+        colortermsFile = os.path.join(os.environ["OBS_SUBARU_DIR"],"config","hsc","colorterms.py")
+        self.photoCalConf = photoCal.PhotoCalConfig()
+        self.photoCalConf.colorterms.load(colortermsFile)
+
+    def testHSCColorterms(self):
+        """Test that the colorterm libraries are formatted correctly"""
+        refBands = ["g","r","i","z","y"]
+        hscBands = ["g","r","i","z","y"]
+        for band in hscBands:
+            ct = self.photoCalConf.colorterms.getColorterm(band, photoCatName="hsc") # exact match
+            self.assertTrue(ct.primary in refBands)
+            self.assertTrue(ct.secondary in refBands)
+            self.assertTrue(isinstance(ct.c0, numbers.Number))
+            self.assertTrue(isinstance(ct.c1, numbers.Number))
+            self.assertTrue(isinstance(ct.c2, numbers.Number))
+
+    def testHSCColortermG(self):
+        """Test that the 'g' colorterm matches the values in config/colorterms.py""" 
+        ctg = self.photoCalConf.colorterms.getColorterm("g", photoCatName="hsc") # exact match
+        self.assertEqual(ctg.primary, "g")
+        self.assertEqual(ctg.secondary, "g")
+        self.assertEqual(ctg.c0, 0.0)
+        self.assertEqual(ctg.c1, 0.0)
+        self.assertEqual(ctg.c2, 0.0)
+
+    def testSDSSColorterms(self):
+        """Test that the colorterm libraries are formatted correctly"""
+        sdssBands = ["g","r","i","z","y"]
+        hscBands = ["g","r","i","i2","z","y","N816","N921"]
+        for band in hscBands:
+            ct = self.photoCalConf.colorterms.getColorterm(band, photoCatName="sdss") # exact match
+            self.assertTrue(ct.primary in sdssBands)
+            self.assertTrue(ct.secondary in sdssBands)
+            self.assertTrue(isinstance(ct.c0, numbers.Number))
+            self.assertTrue(isinstance(ct.c1, numbers.Number))
+            self.assertTrue(isinstance(ct.c2, numbers.Number))
+
+    def testHSCColortermG(self):
+        """Test that the 'N816' colorterm matches the values in config/colorterms.py""" 
```

Comment:
Can't you read the comparison values in from that file? Instead of hardcoding them here. If you need them to be the same that is.